exercises/fizz-buzz/fizz-buzz.R

```diff
@@ -0,0 +1,2 @@
+fizz_buzz <- function(input){
+}
```

Comment:
Since I can't comment on the empty `lintr.R` file below: What's the argument for including it? As far as I understand the situation, it is likely an artefact from preparing the PR? If so, please delete it.